public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -56,10 +60,12 @@ type Props = {
 export const AlertRuleForm = ({ existing, prefill }: Props) => {
   const styles = useStyles2(getStyles);
   const dispatch = useDispatch();
+  const history = useHistory();
```

Comment:
(I thought we were supposed to use `locationService` instead for this kind of thing?)

Reply:
Yep, good catch!